website/src/panel-layout.tsx

```diff
@@ -183,19 +158,18 @@ export const PanelLayout = (p: {
   return (
     <div className={css.wordPanelContent}>
       <>{panel}</>
-      {commentsPanel === false && (
-        <Button type="button" onClick={handleComment}>
-          Comment
-        </Button>
-      )}
-      {commentsPanel === true && (
+      {isCommenting ? (
         <SubtleButton
           type="button"
-          onClick={handleComment}
+          onClick={() => setIsCommenting(!isCommenting)}
           className={css.buttonSpacing}
         >
           Discard
         </SubtleButton>
+      ) : (
+        <Button type="button" onClick={() => setIsCommenting(!isCommenting)}>
+          Comment
+        </Button>
```

Comment:
Does this need to also post the comment?

Reply:
I don't believe so, the comment is posted within the CommentPanel component!

Reply:
OH this opens the panel I get it now